bower.json

```diff
@@ -18,7 +18,8 @@
     "iron-scroll-target-behavior": "^v2.0.0",
     "px-icon-set": "^2.0.0",
     "px-dropdown": "^4.0.0",
-    "vaadin-grid": "^4.0.0"
+    "vaadin-grid": "master",
```

Comment:
revert this line, https://github.com/vaadin/vaadin-grid/releases/tag/v4.1.0-alpha5 was just released

Reply:
Done.